config/admin_ranks.txt

```diff
@@ -43,15 +45,13 @@ Game Master		= +EVERYTHING *EVERYTHING
 Lazy Master 	= +EVERYTHING -AUTOLOGIN *EVERYTHING
 Three Dog		= -@ -ADMIN +DJ
 
-Head Coder      = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+Scribe 			= +SERVER +DEBUG +VAREDIT +SPAWN -AUTOLOGIN
+Head Scribe 	= +@ +POLL -AUTOLOGIN
 
-Initiate		= +ADMIN +BAN +REJUV 
-Aspirant 		= +ADMIN +BAN +REJUV +SPAWN +SOUND
-Knight 			= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN +VAREDIT
-Paladin			= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +DJ +POLL +VAREDIT
-Head Paladin	= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
-Scribe 			= +STEALTH +POSSESS +REJUV +BUILD +DEBUG +VAREDIT +SOUND +SPAWN +DJ +POLL -AUTOLOGIN
-Head Scribe 	= +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL -AUTOLOGIN
-Elder			= +EVERYTHING
-Host 			= +EVERYTHING *EVERYTHING
+Initiate		= +ADMIN
+Aspirant 		= +@ +BAN
+Knight 			= +@ +SPAWN +FUN +VAREDIT
```

Comment:
Can you confirm that Manage Job Slots is available to Knights?

Reply:
![Bulle-check](https://user-images.githubusercontent.com/31828846/55099739-2ebf3c00-50b8-11e9-94d0-3cd4aea0a108.png)